src/utils.js

```diff
@@ -12,6 +12,16 @@ export function escapeRegExp(str: string) {
   return str.replace(/[-[\]/{}()*+?.\\^$|]/g, "\\$&");
 }
 
+export function negativeRegExp(negationFormat: string) {
+  switch (negationFormat) {
+    case 'parentheses':
+      return /\(/;
+    case 'negative':
```

Comment:
The prop options are `minus` and `parentheses `. 
Also as we just have two options for negationFormat and minus is default, also we have added type check for that. Using ternary can reduce the overall size.
```js
export function negativeRegExp(negationFormat: string) {
  return negationFormat === 'parentheses' ? /\(/ : /-/
}
```

Reply:
Good catch, fixed.